AzFilesHybrid/readme.md

```diff
@@ -1,5 +1,23 @@
 # AzFilesHybrid
-
 ## Overview
-
-The AzFilesHybrid PowerShell module provides cmdlets for deploying and configuring Azure Files. It offers cmdlets for domain joining storage accounts to your on-premises Active Directory, configuring your DNS servers, and troubleshooting  authentication issues.
+The AzFilesHybrid PowerShell module provides cmdlets for deploying and configuring Azure Files. It offers cmdlets for domain joining storage accounts to your on-premises Active Directory, configuring your DNS servers, and troubleshooting  authentication issues. 
+## Installation
+### Prerequisites
+[PowerShell 5.1+](https://learn.microsoft.com/en-us/powershell/scripting/install/installing-powershell-on-windows?view=powershell-5.1) or [PowerShell 7+](https://learn.microsoft.com/en-us/powershell/scripting/install/installing-powershell-on-windows?view=powershell-7.5#install-powershell-using-winget-recommended)) \
+[.NET Framework 4.7.2+](https://dotnet.microsoft.com/en-us/download/dotnet-framework/net472)
+### Install From PSGallery (recommended)
+Open a PowerShell7 terminal \
+Paste in the following Code snippet and run \
+`Install-Module -Name AzFilesHybrid` \
+[For other installation options and the latest version](https://www.powershellgallery.com/packages/AzFilesHybrid/)
```

Comment:
The backslashes are not the most natural way to skip lines in markdown. Let's maybe do this?

````suggestion
Run the following in a PowerShell window
    
```powershell
Install-Module -Name AzFilesHybrid
```
````